@types/gql.d.ts

```diff
@@ -45,42 +53,29 @@ type UpdateProjectArgs = {
 };
 
 type ProjectQuery = {
-    _id?: string;
     id: string;
-    number: string;
     userID: string;
     projectID: string;
     number: number;
-    depth: number;
     complexity: number;
+    depth: number;
     tokens: number;
     success: boolean;
     timestamp: number;
+    loggedOn: number;
     latency?: number;
 };
 
 type CreateProjectQueryArgs = {
     projectQuery: {
         projectID: string;
         number: string;
-        depth: number;
         complexity: number;
+        depth: number;
         tokens: number;
         success: boolean;
         timestamp: number;
-        latency?: number;
-    };
-};
-
-type UpdateProjectQueryArgs = {
```

Comment:
Where did UpdateProjectQueryArgs go?

Reply:
This isn't me, this is from @donjewey clean up of the backend.

Reply:
yes, there is no need to update project queries' information. only added this in the beginning as a means of being thorough, but now it is not necessary